### PR TITLE
updates on desertion

### PR DESCRIPTION
**Pull request by anperez78**

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -354,28 +361,35 @@ router.post('/v1/reason/adultery', function(req, res) {
...
     year: req.body.desertionYear || '2014'
   };
-  var details = req.body.desertionDetails ||
-    'They left and I haven\'t seen them since.';
-  var hasLivedApart = req.body.hasLivedApart || 'yes';
-
-  req.session.facts.details = details;
   req.session.facts.desertionDate = desertionDate;
-  req.session.facts.hasLivedApart = hasLivedApart;
+  res.redirect('/v1/reason/desertion/leave-apart-for-2-years');
```
Comment (sophiedrake) [inline app/routes.js]: 'live' or 'lived'

**Comment by sophiedrake** [comment]
I think anywhere you have something similar to `leave-apart-for-2-years` should be `lived-apart-for-2-years`

File: app/views/v1/exit/desertion-time-period-fail.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% extends "exit.html" %}
+
+{% block heading %}
+You cannot continue with your application
+{% endblock %}
+
+{% block exit_reason %}
+  <p class="text">
+    You must have lived apart for total period of more than 2 years.
```
Comment (sophiedrake) [inline app/views/v1/exit/desertion-time-period-fail.html]: `for a total period`

File: app/views/v1/reason/desertion/leave-date.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+{% block question %}
+  Supporting information
+{% endblock %}
+
+{% block form %}
+
+<fieldset class="form-group">
+  <div class="form-date">
+    <label class="form-label-bold">When did your husband/ wife desertion?</label>
```
Comment (sophiedrake) [inline app/views/v1/reason/desertion/leave-date.html]: `When did your husband/ wife leave?`

File: app/views/v1/reason/desertion/leave-apart-for-2-years.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{% extends "question.html" %}
+
+{% block question %}
+  Have you lived apart for a period of 2 years?
+{% endblock %}
+
+{% block form %}
+
+<fieldset class="form-group inline">
+    <label class="form-label-bold">You may have got back together/reunited
```
Comment (sophiedrake) [inline app/views/v1/reason/desertion/leave-apart-for-2-years.html]: `class="form-label text"`

File: app/routes.js (hunk 1 above)
Comment (anperez78) [inline app/routes.js]: 👍

File: app/views/v1/exit/desertion-time-period-fail.html (hunk 2 above)
Comment (anperez78) [inline app/views/v1/exit/desertion-time-period-fail.html]: 👍

File: app/views/v1/reason/desertion/leave-date.html (hunk 3 above)
Comment (anperez78) [inline app/views/v1/reason/desertion/leave-date.html]: 👍

File: app/views/v1/reason/desertion/leave-apart-for-2-years.html (hunk 4 above)
Comment (anperez78) [inline app/views/v1/reason/desertion/leave-apart-for-2-years.html]: 👍

**Pull request merged**